tests/unittest_nodes_position.py

```diff
@@ -0,0 +1,152 @@
+import textwrap
+from typing import List
+
+from astroid import builder, nodes
+
+
+class TestNodePosition:
+    """Test node ``position`` attribute."""
+
+    @staticmethod
+    def test_position_class() -> None:
+        """Position should only include keyword and name.
+
+        >>> class A(Parent):
+        >>> ^^^^^^^
+        """
+        code = textwrap.dedent(
+            """
+        class A:  #@
+            ...
+
+        class B(A):  #@
+            pass
+
+        class C:  #@
+            '''Docstring'''
+
+            class D:  #@
+                ...
+
+        class E:  #@
```

Comment:
Shall we add one test with a decorator? I tested and it works fine currently, but it might be good to have it in there as well.